environment/plaintext/docker-compose.yml

```diff
@@ -33,7 +36,7 @@ services:
       KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
       KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: BROKER:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
       KAFKA_LISTENERS: BROKER://:9092,PLAINTEXT_HOST://:29092
-      KAFKA_ADVERTISED_LISTENERS: BROKER://broker:9092,PLAINTEXT_HOST://localhost:29092
+      KAFKA_ADVERTISED_LISTENERS: BROKER://broker:9092,PLAINTEXT_HOST://broker:29092
```

Comment:
Only reason I made the change is it appeared as if it was a mistake initially. Since it appears to be on-purpose for localhost testing, I'll revert the change back.

Reply:
not a mistake :)

Reply:
fixed :)